Main.java

```diff
@@ -25,39 +28,56 @@ public static void main(String[] args) {
 
 		// 3:「3年後の今日が何曜日なのか」を計算して、コンソール画面に表示してみよう。
 		LocalDateTime now = LocalDateTime.now();
-		LocalDateTime localDateTime = now.plusYears(3);
-		DayOfWeek week = localDateTime.getDayOfWeek();
+		LocalDateTime threeYearsLater = now.plusYears(3);
+		DayOfWeek week = threeYearsLater.getDayOfWeek();
 		System.out.println(week);
 
 		System.out.println("------------");
 
 		// 4:「引数が0なら偽。引数が1なら真」とコンソール画面に表示してみよう。
-		int bit = new java.util.Random().nextInt(2);
-		System.out.println(judgment(bit));
+		int bit = new Random().nextInt(2);
+		String bitAnswer = judgement(bit);
+		String bitException = "Error: もう一度繰り返します。";
+		int count = 1;
+		while (bitException.equals(bitAnswer)) {
+			count++;
+			if (count == 3) {
+				System.out.println("Error: 正しく処理できませんでした。強制終了します。");
+				System.exit(0);
+			}
+			System.out.println(bitException);
+			System.out.println("------------");
+			bit = new Random().nextInt(2);
+			bitAnswer = judgement(bit);
+		}
+
+		System.out.println(bitAnswer);
 
 		System.out.println("------------");
 
 		// 5:String型のListかMapを作成して、その内容をコンソール画面に表示してみよう。
-		ArrayList<String> arrayString = new ArrayList<String>();
-		arrayString.add("2回目の課題");
-		System.out.println(arrayString.get(0));
+		List<String> homeworkList = new ArrayList<String>();
+		homeworkList.add("2回目の課題");
+		System.out.println(homeworkList.get(0));
 
 		System.out.println("------------");
 
 		// 6:（自由課題）任意のプログラムで例外を起こしてみる。
 		try {
 			System.out.println("任意の数値を入力して下さい");
-			int number = new java.util.Scanner(System.in).nextInt();
+			int number = new Scanner(System.in).nextInt();
 			System.out.println(number);
 		} catch (InputMismatchException e) {
 			System.out.println("入力されたものは適切ではありません、数値を入力して下さい");
 		}
 	}
 
-	public static boolean judgment(int i) {
+	public static String judgement(int i) {
 		if (i == 0) {
-			return false;
+			return "偽";
+		} else if (i == 1) {
+			return "真";
 		}
-		return true;
+		return "Error: もう一度繰り返します。";
```

Comment:
Exceptionを扱うのであれば
```suggestion
		throw new Exception("0か1を入力してください);
```
とするのがよいです。
伴ってこのメソッドを呼び出しているところのwhile文の修正が必要になります。
うまく直せそうになかったら相談くださいね！